src/config.js

```diff
@@ -10,10 +10,10 @@ const api = axios.create({
 api.interceptors.request.use((config) => {
   const configuration = config;
 const authToken = loadAuthToken()
-const {booksItemCount, listsItemCount, pageNumber, sortBy, listIds} = config;
+const {booksItemCount, listsItemCount, pageNumber, sortBy, listIds, bookQuery} = config;
```

Comment:
@zorro1rr we got some formatting issues with this file, is your eslint and prettier working okay?